save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -62,4 +64,69 @@ class DemoManagerController(
                 .retrieve()
                 .toBodilessEntity()
         }
+        .flatMap {
+            projectService.findByNameAndOrganizationNameAndStatusIn(projectName, organizationName, setOf(ProjectStatus.CREATED)).toMono()
+        }
+        .switchIfEmptyToNotFound {
+            "Could not find project $projectName in organization $organizationName"
+        }
+        .map { project ->
+            lnkProjectGithubService.saveIfNotPresent(project, demoToolRequest.ownerName, demoToolRequest.repoName)
```

Comment:
why `saveIfNotPresent` -- maybe return an error in case of duplicate?

Reply:
Should be done probably, agree
Will do it

Reply:
Done in https://github.com/saveourtool/save-cloud/pull/1668/commits/b14eb0b1a75f17a8cb689f893674009195285fe4